cypress/fixtures/connectors/sftp_sink_0.1.json

```diff
@@ -0,0 +1,206 @@
+{
+  "channels": {
+    "stable": {
+      "shard_metadata": {
+        "annotations": {
+          "trait.camel.apache.org/container.request-cpu": "0.20",
+          "trait.camel.apache.org/container.request-memory": "128M",
+          "trait.camel.apache.org/deployment.progress-deadline-seconds": "30"
+        },
+        "connector_image": "quay.io/rhoas/cos-connector-sftp:main-f5f1f4b5a6e078b016886b292e81a19aad588cdd",
+        "connector_revision": 82,
+        "connector_type": "sink",
+        "consumes": "application/octet-stream",
+        "error_handler_strategy": "stop",
+        "kamelets": {
+          "adapter": {
+            "name": "sftp-sink",
+            "prefix": "sftp"
+          },
+          "kafka": {
+            "name": "cos-kafka-source",
+            "prefix": "kafka"
+          }
+        },
+        "operators": [
+          {
+            "type": "camel-connector-operator",
+            "version": "[1.0.0,2.0.0)"
+          }
+        ],
+        "produces": "application/octet-stream"
+      }
+    }
+  },
+  "connector_type": {
+    "capabilities": ["data_shape", "error_handler", "processors"],
+    "channels": ["stable"],
+    "description": "Send data to an SFTP Server.",
+    "icon_href": "TODO",
+    "id": "sftp_sink_0.1",
+    "kind": "ConnectorType",
+    "labels": ["sink"],
+    "name": "SFTP sink",
+    "schema": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "sftp_connection_host",
+        "sftp_connection_port",
+        "sftp_username",
+        "sftp_password",
+        "sftp_directory_name",
+        "kafka_topic"
+      ],
+      "properties": {
+        "sftp_connection_host": {
+          "title": "Connection Host",
+          "description": "The hostname of the FTP server",
+          "type": "string"
+        },
+        "sftp_connection_port": {
+          "title": "Connection Port",
+          "description": "The port of the FTP server",
+          "type": "string",
+          "default": "22"
+        },
```

Comment:
Making the default value coherent with the field type (`string`) solves the issue.

Current default [is a number](https://github.com/bf2fc6cc711aee1a0c2a/cos-manifests/blob/914e6be67dfd77883f34d59f6fd31396a39e8924/connectors/cos-fleet-catalog-camel/connector-catalog-camel-storage/sftp_sink_0.1.json#L56).